html/components/_masthead.scss

```diff
@@ -348,12 +348,27 @@
   padding: $sprk-masthead-accordion-item-padding;
   text-decoration: none;
 
+  // Button User Agent Resets
+  min-width: 100%;
+  background: $sprk-masthead-accordion-item-background;
+  border: $sprk-masthead-accordion-item-border;
+  text-align: $sprk-masthead-accordion-item-text-align;
+
   &:hover,
   &:active {
     background-color: $sprk-masthead-accordion-active-background-color;
     color: $sprk-masthead-accordion-active-font-color;
     text-decoration: none;
   }
+
+  // Polyfill for focus-visible
+  &:focus:not(.focus-visible) {
+    outline: none;
+  }
+}
+
+.sprk-c-MastheadAccordion__summaryIcon {
```

Comment:
Following class naming convention this would be `.sprk-c-MastheadAccordion__icon`.

Reply:
the __icon class already exists.
This is the icon specifically inside the summary, not the icon in the masthead

what's also weird about `sprk-c-MastheadAccordion__icon` is that it's only being used in angular 
![image](https://user-images.githubusercontent.com/4342363/79149440-2c985000-7d95-11ea-8c39-87e966ce0db3.png)